packages/php-db-import-export/src/Backend/Snowflake/SnowflakeException.php

```diff
@@ -38,6 +38,14 @@ public static function covertException(Throwable $e): Throwable
             );
         }
 
+        if (preg_match('/ \'([^\']*)\' cannot be inserted because it\'s bigger than column size/', $e->getMessage(), $output_array) === 1) {
```

Comment:
```suggestion
        if (preg_match('/ \'(.*)\' cannot be inserted because it\'s bigger than column size/', $e->getMessage(), $output_array) === 1) {
```

Reply:
https://regex101.com/r/qR2EgL/1

Reply:
upravil sem aj pro předchozí https://github.com/keboola/storage-backend/pull/68/commits/09eb34afccf9392041934012a8cc0c974f799b06

Reply:
Tak uprav ještě prosím testy :)

Reply:
Testy testujů reálné hlášky ze SNFLK a s tůto úpravů projdů.

Reply:
Jo, já myslel tam přidat ten apostrof, který předtím neprošel

Reply:
Máš k tomu reálnů error message ze SNFLK?

Reply:
No,nemam, ale zrovna u string "libovolný string" být apostrof může... 
https://my.papertrailapp.com/groups/23635/events?q=bigger+than+column+size&focus=1593672146876502022&selected=1593672146876502022

Reply:
přidal sem tam test se shity hodnotú